installer/templates/phx_assets/package.json

```diff
@@ -11,16 +11,13 @@
     "topbar": "^0.1.4"<% end %>
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "babel-loader": "^8.0.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.4.2",
-    "hard-source-webpack-plugin": "^0.13.1",
-    "mini-css-extract-plugin": "^0.9.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "terser-webpack-plugin": "^2.3.2",
-    "webpack": "4.41.5",
-    "webpack-cli": "^3.3.2"
+    "copy-webpack-plugin": "^9.0.0",
```

Comment:
I do not see `copy-webpack-plugin` beeing used anymore within `webpack.config.js`.
Means it could be removed here.

But by removing `copy-webpack-plugin` I think in `installer/lib/phx_new/single.ex` copy path of `robots.txt`, `phoenix.png` and `favicon.ico` should be changed from `assets/static/` to `priv/static/`.

With this change `/priv/static` should get removed in the `.gitignore`.

Reply:
moving them over to `priv/static` might make sense, but it depends upon what experience @chrismccord wants here. If all web assets should be in the `assets/` directory then copied over at build time I can bring back the copy rule.

Reply:
We need to support copying assets from `assets/static/` to `priv/static`, so we'lll need copy-webpack unless webpack can do something like this out of the box now. This allows folks to put all assets inside the assets directory, and build them into private/static, which is especially important for our digest task, which will digest the copied/built assets

Reply:
Then it would be necessary to bring it back in.

I'm not aware of any simpler out of the box webpack solution.

One thing I once encountered but haven't evaluated in detail then,
`copy-webpack-plugin` caused to copy (or touch all copy files) if one of them canged. Means live reload reloads x times (or logged to reload).
But could be, that this was only true fot a certain version of the updated plugin.

Reply:
@cw789 I'm working on putting it back right now, the API changed with the version bump so I just have to ensure that it's doing everything we want.